bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        use_stdin=True,
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+): (?P<message>.*)')
```

Comment:
Invalid regex.
It prints `Error in `filename`: Near line `line_number` last key parsed `key` : Unexpected Key parsed `key``.